anaron/02.CSS/CSS_02_Cards.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<HTML>
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Спрайты</TITLE>
+    <style type="text/css">
+        .k {
+            width: 80px;
+            height: 100px;
+            display: inline-block;
+            vertical-align: top;
+            margin-right: -20px;
+        }
+
+    </style>
+</HEAD>
+
+<BODY>
+    <div class="k 3"> <img width="75" height="75" src="3.png" alt="3">

```

Comment:
interesting approach, but in this task need use one picture instead a group, like we did on our webinar